### ISSUE TITLE
Clean up scss comments

### ISSUE THREAD

**Issue by joolswood** [AusDTO/gov-au-ui-kit#444]
We need to go through the scss partials and clean up any remaining comments, following the Design Guide content refactoring.

**Comment by klepas** [comment]
@dominikwilkowski What do you think the value of this is wrt. time and our desire to move the focus to 2.x? I’m inclined to close this, unless there are SASS partials now that have comment sections which make zero sense.

Thoughts?

**Comment by klepas** [comment]
Agreed to wait until some 2.x stuff is done, but keeping this open to go back to our SASS to clean them up for the 1.x codebase later.
